pysd/py_backend/model.py

```diff
@@ -709,12 +711,11 @@ def set_components(self, params, new=False):
         >>> br = pandas.Series(index=range(30), values=np.sin(range(30))
         >>> model.set_components({'birth_rate': br})
 
-
         """
-        # TODO: allow the params argument to take a pandas dataframe, where
+        #  TODO: allow the params argument to take a pandas dataframe, where
```

Comment:
Please, remove the extra spaces before TODOs, this would make more difficult to filter TODO comments using a searcher